.github/workflows/rtds_build_and_push_dev.yaml

```diff
@@ -57,6 +58,7 @@ jobs:
           while ! docker exec rtdsagent-2 nc localhost 4343 ; do sleep 1; done;
           docker cp rtdshub:/install/hvr.lic tests/hvr.lic
           pytest tests/test_clean_install.py
+          pytest tests/test_user.py
```

Comment:
Better to run them together (single pytest call with two files listed), this way it shows all results in one output.